### ISSUE TITLE
Same name of Group and same name of Assignments should not be their

### ISSUE THREAD

**Issue by ryuk156** [CircuitVerse/CircuitVerse#1051]
**Describe the bug**
When we create a group of name eg : yash15601 the one group is created now create the another group of same name i.e yash15601 ,it will create the group of same group
Same in Assignments :,When we create a Assignments of name eg : myfistassignments the one Assignment is created now create the another Assignments of same name i.e myfistassignment ,it will create Assignments of same name

**Expected behavior**
 Their should not exists the same name of Assignments  and group
**Screenshots**
Group:
![image](https://user-images.githubusercontent.com/49101492/76143878-f7ffed00-60a0-11ea-8c4f-8e1df4a79cee.png)

Assignments:
![image](https://user-images.githubusercontent.com/49101492/76143932-7a88ac80-60a1-11ea-96cc-8026e771a988.png)

**Are you working on this issue? (Yes/No)**
Yes

**Comment by nitin10s** [comment]
Didn't understand what is the expected behaviour?

**Comment by Ananthavijay** [comment]
Group,Project,Assignment names should be unique

**Comment by yash15601** [comment]
@nitin10s  As said by  @Ananthavijay

**Comment by nitin10s** [comment]
Great observation! :)

**Comment by yash15601** [comment]
@nitin10s  thanks 😂

**Comment by satu0king** [comment]
This is not really much of an issue. Any fix should consider the following things
1. It should be efficient
1. It should not break current assignments/groups etc

**Comment by yash15601** [comment]
@satu0king  like having same name of group will create problem for user to identify the in which he/she added related  members  a part from this about efficiency  and breaking assignment /group  will fix this soon

**Comment by vedant-jain03** [comment]
I am working on this issue.

**Comment by ayan-biswas0412** [comment]
> What I was thinking is if a user is creating an assignment/group with the same name instead of showing the message `Same name group/assignment already exist` we can just confirm that if he/she wanted to continue. We can popup something like `${name} assignment/group already exist, are you sure want to create a new one?`. In this way, it is just in the user's hands.
> 
> WDYT @gr455 @tachyons @ayan-biswas0412 @ryuk156 ?

yah I think this check should be there.

**Comment by Prerna-0202** [comment]
> What I was thinking is if a user is creating an assignment/group with the same name instead of showing the message `Same name group/assignment already exist` we can just confirm that if he/she wanted to continue. We can popup something like `${name} assignment/group already exist, are you sure want to create a new one?`. In this way, it is just in the user's hands.
> 
> WDYT @gr455 @tachyons @ayan-biswas0412 @ryuk156 ?

I also agree this check should be there. I am working on this issue and will raise a pr soon.

**Comment by chaitanyahere** [comment]
Hello ! Is this issue assigned to someone.
I can fix this issue can you please assign me, if its not assigned!

**Comment by Prerna-0202** [comment]
@chaitanyahere Go ahead 👍
